cost-analyzer/charts/prometheus/values.yaml

```diff
@@ -865,11 +865,11 @@ server:
 
   ## Security context to be added to server pods
   ##
-  securityContext:
-    runAsUser: 1001
-    runAsNonRoot: true
-    runAsGroup: 1001
-    fsGroup: 1001
+  securityContext: {}
```

Comment:
Any reason you're commenting out the `securityContext` in these subcharts? Will this affect existing users currently requiring these permission?

Reply:
The reason these are commented out is because with the move in order of precedence, if they were not then they would apply. Currently, they do not, so these changes are basically no-ops so long as the global level securityContext is defined. In other words, these changes produce the same effect as what users experience today. It's only *IF* they decide to use more granular settings (which requires explicit action) will that result change.

Reply:
Oh I see! Currently, this prometheus-server is already taking the global security context:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/f481c59dc79e7eaf79dc6e607511d9eb52b1d67c/cost-analyzer/charts/prometheus/templates/server-deployment.yaml#L63-L65

However when changing the order of precedence across all securitycontexts, we need to comment out highest granularity (and currently unused) securitycontext for prometheus-server so that it continues to take the global security context.

Reply:
Yes, and those changes have been made in this PR.